src/jenkins/ReleaseMetricsData.groovy

```diff
@@ -58,7 +61,7 @@ class ReleaseMetricsData {
                                         ],
                                         [
                                                 match_phrase: [
-                                                        "component": "${component}"
+                                                        "component.keyword": "${component}"
```

Comment:
you don't need this if `component` field is of `keyword` type. 
Was there any change in results?

Reply:
Yes the type is text. This is the mapping:
```
        "component": {
          "type": "text",
          "fields": {
            "keyword": {
              "type": "keyword",
              "ignore_above": 256
            }
          }
        },
```
Without `keyword` the the results return everything that has opensearch in it.

Reply:
Ideally the field should have been keyword type. I'm okay if mapping cannot be updated.

Reply:
Right! I am just retrieving data from that index. @prudhvigodithi might be able to add more.